package/crds/ansible.crossplane.io_ansibleruns.yaml

```diff
@@ -41,9 +40,13 @@ spec:
             properties:
               deletionPolicy:
                 default: Delete
-                description: DeletionPolicy specifies what will happen to the underlying
+                description: 'DeletionPolicy specifies what will happen to the underlying
```

Comment:
Are these changes brought in automagically with crossplane upgrade?

Reply:
Yes